mps_youtube/helptext.py

```diff
@@ -79,6 +79,8 @@ def helptext():
     {2}dvupl <username>{1} - download user's YouTube playlists (video).
     {2}dlurl <url or id>{1} download a YouTube video by url or video id.
     {2}playurl <url or id>{1} play a YouTube video by url or id.
+    {2}browserplay <url or id>{1} play a YouTube video by url or id in browser.
```

Comment:
I thought the requested feature was to search a video from mpsyt and then open the video inside your browser like
```
/the nights
browser 1-2
```
which would open the results 1 and 2 of 'the nights' search in the browser.